dev/io.openliberty.persistence.3.1.thirdparty/bnd.bnd

```diff
@@ -132,11 +132,7 @@ Include-Resource: \
   @${repo;org.eclipse.persistence:org.eclipse.persistence.jpa.jpql;${eclFullVersion};EXACT}!/!(META-INF/maven/*|module-info.class),\
   @${repo;org.eclipse.persistence:org.eclipse.persistence.jpa.modelgen.processor;${eclFullVersion};EXACT}!/!(META-INF/maven/*|module-info.class),\
   @${repo;org.eclipse.persistence:org.eclipse.persistence.json;${eclFullVersion};EXACT}!/!(META-INF/maven/*|module-info.class),\
-  org/eclipse/persistence/asm=${bin}/org/eclipse/persistence/asm,\
-  org/eclipse/persistence/dynamic=${bin}/org/eclipse/persistence/dynamic,\
-  org/eclipse/persistence/internal/jpa=${bin}/org/eclipse/persistence/internal/jpa,\
-  org/eclipse/persistence/mappings/converters=${bin}/org/eclipse/persistence/mappings/converters,\
-  org/eclipse/persistence/jpa/jpql/parser=${bin}/org/eclipse/persistence/jpa/jpql/parser
+  org/eclipse/persistence/mappings/converters=${bin}/org/eclipse/persistence/mappings/converters
```

Comment:
The overlay associated with `mappings/converters` also needs to be remove; see https://github.com/OpenLiberty/open-liberty/pull/27872

Reply:
done